manifests/modules/automation/controlplanes/crossplane/compositions/claim/claim.yaml

```diff
@@ -1,11 +1,13 @@
 apiVersion: awsblueprints.io/v1alpha1
-kind: RelationalDatabase
+kind: DynamoDBTable
 metadata:
-  name: catalog-composition
-  namespace: catalog
+  name: items
```

Comment:
This name will have to align with the cluster name, so should be something like `${EKS_CLUSTER_NAME}-carts-crossplane`

Reply:
Updated